lib/Github/Api/GraphQL.php

```diff
@@ -16,15 +16,19 @@ class GraphQL extends AbstractApi
     
     /**
      * @param string $query
+     * @param array $variables
      *
      * @return array
      */
-    public function execute($query)
+    public function execute($query, array $variables = null)
```

Comment:
Change the default value to an emtpy array, this way each possible value (user input or default) is the same type